src/CONFIG.js

```diff
@@ -49,11 +49,17 @@ export default {
     EXPENSIFY: {
         // Note: This will be EXACTLY what is set for EXPENSIFY_URL whether the proxy is enabled or not.
         EXPENSIFY_URL: expensifyURL,
-        SECURE_EXPENSIFY_URL: secureURLRoot,
         NEW_EXPENSIFY_URL: newExpensifyURL,
-        URL_API_ROOT: expensifyURLRoot,
-        STAGING_EXPENSIFY_URL: stagingExpensifyURL,
-        STAGING_SECURE_EXPENSIFY_URL: stagingSecureExpensifyUrl,
+
+        // The PRIMARY API is the API used by most environments, but staging, where we use STAGING (bellow)
+        // The "staging toggle" in settings toggles between PRIMARY and STAGING APIs
+        // We can think of PRIMARY as PROD, because on both STAGING and PROD it points to the production address
+        // On DEV this can be configured through ENV settings and can be a proxy or ngrok address (defaults to PROD)
+        // Usually you don't need to use this URL directly - prefer `ApiUtils.getApiRoot()`
+        PRIMARY_API_ROOT: expensifyURLRoot,
+        PRIMARY_SECURE_API_ROOT: secureURLRoot,
+        STAGING_API_ROOT: stagingExpensifyURL,
+        STAGING_SECURE_API_ROOT: stagingSecureExpensifyUrl,
```

Comment:
We have this thread, that's hidden due to the updated code comment: https://github.com/Expensify/App/pull/15178#discussion_r1126937268